tests/e2e/assets/test_subnet_evm_genesis.json

```diff
@@ -21,18 +21,6 @@
       "minBlockGasCost": 0,
       "maxBlockGasCost": 1000000,
       "blockGasCostStep": 200000
-    },
```

Comment:
We could. You think it's more clear if we set it back so it's visible that we are setting those variables to zero?

If you prefer that, then I'd appreciate you create a ticket for that so that we don't forget.

Reply:
not preference for this